#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_check_tcp_fix_help_escape.dpatch by Jan Wagner <waja@cyconet.org>

From 22dfca20fd83da319ec23160e0092a602b6c1eea Mon Sep 17 00:00:00 2001
From: Sebastian Herbszt <herbszt@gmx.de>
Date: Sun, 30 Nov 2014 23:57:47 +0100
Subject: [PATCH] check_tcp: fix help description regarding escape option
Origin: upstream, https://github.com/monitoring-plugins/monitoring-plugins/commit/22dfca20fd83da319ec23160e0092a602b6c1eea.patch

The help description should print '\\' for backslash not just '\'.

@DPATCH@

Signed-off-by: Sebastian Herbszt <herbszt@gmx.de>
---
 plugins/check_tcp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/check_tcp.c b/plugins/check_tcp.c
index fc0adba..63f9fd9 100644
--- a/plugins/check_tcp.c
+++ b/plugins/check_tcp.c
@@ -643,7 +643,7 @@ print_help (void)
 	printf (UT_IPv46);
 
 	printf (" %s\n", "-E, --escape");
-  printf ("    %s\n", _("Can use \\n, \\r, \\t or \\ in send or quit string. Must come before send or quit option"));
+  printf ("    %s\n", _("Can use \\n, \\r, \\t or \\\\ in send or quit string. Must come before send or quit option"));
   printf ("    %s\n", _("Default: nothing added to send, \\r\\n added to end of quit"));
   printf (" %s\n", "-s, --send=STRING");
   printf ("    %s\n", _("String to send to the server"));
